modules/standard/IO.chpl

```diff
@@ -793,37 +792,31 @@ Format strings in Chapel consist of:
 # Specifiers
 ++++++++++++
 
-Simply display how many digits to use when printing a floating-point number by
-using the # sign to stand for digits. The fractional portion of the number will
-be rounded appropriately and extra space will be made if the integer portion is
-too small:
+A # specifiers must be enclosed in %{} syntax, for example ``%{#}`` is the
+shortest one, and ``%{#.#}`` is a more typical one. The integer portion of the
+number will be padded out to match the number of ``#``s before the decimal
+point, and the number of ``#``s after the decimal point indicate how many
+digits to print after the decimal point. In other words, display how many
+digits to use when printing a floating-point number by using the # sign to
+stand for digits. The fractional portion of the number will be rounded
+appropriately and extra space will be made if the integer portion is too small:
 
 .. code-block:: chapel
 
-  writef("n:###.###\n", 1.2349);
+  writef("n:%{###.###}\n", 1.2349);
        // outputs:
        // n:  1.235
 
 This syntax also works for numbers without a decimal point by rounding them
 appropriately.
 
-A # specifier must start with a '#'. For example, if we start with a '.'
-it will interpret the . as a literal, as in:
+A # specifier must start with a ``.``.
 
 .. code-block:: chapel
 
-  writef(".###\n", 0.777);
+  writef("%{.##}\n", 0.777);
        // outputs:
-       // .  1
```

Comment:
Is this correct? If '.' may appear as part of a template format string, then I would expect the rounded result to be ".78".